KMB/kmb_massload.py

```diff
@@ -169,34 +169,46 @@ def process_byline(entry):
         entry['byline'] = helpers.flip_name(entry['byline'])
 
 
-# @todo: update this per new copyright rules - T164568
 def process_license(entry):
     """
-    Identify the license and store back in entry.
+    Identify the license and store the back in entry.
 
     Don't include name/byline if unknown.
     """
+    entry['copyright'] = entry['copyright'].strip()
+    license_text = None
+
     if entry['license']:
         trim = 'http://kulturarvsdata.se/resurser/License#'
         entry['license'] = entry['license'].strip()[len(trim):]
-    # if not copyright = RAÄ and if license='' then, this is probably unfree
+
     if (entry['license'] == 'pdmark') or \
-            (entry['copyright'].strip() == 'Utgången upphovsrätt'):
-        entry['license'] = '{{PD-Sweden-photo}}'
-    elif (entry['license'] == 'by') or (entry['copyright'].strip() == 'RAÄ'):
-        # consider changing this to AND since there might be a cc-by image which isn't from RAA.
-        # Alternatively have another if inside which checks whethere copyright = RAA
-        param = '}}'
-        if (entry['byline'] == '{{unknown}}') or \
-                (entry['byline'] == '{{not provided}}'):
-            pass
-        else:
-            param = '|%s}}' % entry['byline']
-        entry['license'] = '{{CC-BY-RAÄ%s' % param
+            (entry['copyright'] == 'Utgången upphovsrätt'):
+        license_text = '{{PD-Sweden-photo}}'
+    elif entry['license'] == 'by':
+        byline = None
+        template = 'CC-BY-SA-2.5'
+        if entry['byline'] not in ('{{unknown}}', '{{not provided}}'):
+            byline = '|{0}'.format(entry['byline'])
+
+        if entry['copyright'] == 'RAÄ':
+            template = 'CC-BY-RAÄ'
```

Comment:
The images we are uploading are all low resolution. The high resolution ones only exist on CD-ROMs in the archives.

That said the template should probably be updated since that phrasing has changed and it has since been clarified that only Copyright RAÄ is not enough to make it CC-BY (which is what we were told 5 years ago). That clarification/change is also the main reason for this change.